### PR TITLE
[LIB-471] Date fields in models

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

**Comment by dancio** [comment]
how about doc strings?

File: src/models/base.js [hunk 1]
```diff
@@ -67,7 +67,8 @@ export const Meta = stampit()
     * @returns {Object}
     */
     assignProperties(source, target) {
-      return _.assign({}, this.getObjectProperties(source), target);
+      let dateFields = _.mapValues(_.pick(target, ['created_at', 'updated_at', 'executed_at']), (o) =>  new Date(o));
```
Comment (dancio) [inline src/models/base.js]: `const` ?

**Comment by jakubbilko** [comment]
@dancio - u mean an updated one for the whole function?

**Comment by dancio** [comment]
I mean we have doc string for each model which has type definitions e.g: https://github.com/Syncano/syncano-js/blob/release/1.0.0/src/models/codebox.js#L68

**Comment by jakubbilko** [comment]
Right, good call ;)

**Comment by dancio** [comment]
lgtm

**Pull request merged**